docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -11,9 +11,21 @@
 与`GO`或`LOOKUP`等其他查询语句相比，`MATCH`的语法更灵活。`MATCH`语法可以概括如下：
 
 ```ngql
-MATCH <pattern> [<WHERE clause>] RETURN <output>;
+MATCH <pattern> [<clause_1>]  RETURN <output>  [<clause_2>];
 ```
 
+- `pattern`：pattern 的详细说明请参见[模式](../1.nGQL-overview/3.graph-patterns.md)。`MATCH`语句支持匹配一个或多个模式，多个模式之间用英文逗号（,）分隔。例如`(a)-[]->(b),(c)-[]->(d)`。
+
+- `clause_1`：支持`WHERE`、`WITH`、`UNWIND`、`OPTIONAL MATCH`子句，也可以使用`MATCH`作为子句。
+
+- `output`：定义需要返回的输出。可以使用`AS`设置输出的别名。
+
+  !!! compatibility "历史版本兼容性"
+
+        从3.0版本开始，`pattern`支持同时匹配多个 Tag、Edge Type，所以返回属性时，需要额外指定 Tag/Edge type 名称。即从`RETURN 变量名.属性名`改为`RETURN 变量名.Tag.属性名`或`RETURN 变量名.Edge_type.属性名`。
```

Comment:
only tag need specify the tag name， not edge, because  If the type of a variable is edge, 
 `var.propName `  will not raise objections